Frameworks/UIKit/UITextField.mm

```diff
@@ -640,26 +384,8 @@ - (UITextBorderStyle)borderStyle {
 - (void)setBackground:(UIImage*)image {
     _backgroundImage = image;
 
-    // Gap: Need to add using CIImage to set ImageBrush when CGImage is NULL after CIImage is implemented.
-    CGImageRef cgImg = [image CGImage];
-    if (cgImg == NULL) {
-        TraceWarning(TAG, L"cgImg is NULL, need to use CIImage to initialize ImageBrush.");
-        return;
-    }
-
-    if (_borderStyle != UITextBorderStyleRoundedRect) {
-        Microsoft::WRL::ComPtr<IInspectable> inspectableNode(DisplayTexture::GetBitmapForCGImage(cgImg));
-        auto bitmapImageSource = objcwinrt::from_insp<Media::Imaging::BitmapSource>(inspectableNode);
-        Media::ImageBrush imageBrush;
-        imageBrush.ImageSource(bitmapImageSource);
-
-        [_secureModeLock lock];
-        if (_secureTextMode) {
-            _passwordBox.Background(imageBrush);
-        } else {
-            _textBox.Background(imageBrush);
-        }
-        [_secureModeLock unlock];
+    if (self.borderStyle != UITextBorderStyleRoundedRect) {
```

Comment:
comment as to why we need this check? or maybe we just don't do the check and let it look bad until Xaml fixes clipping.